### DISCUSSION TITLE
Any published for the iPhone made with Quasar?

### DISCUSSION THREAD

**Discussion by ostoveland** [quasarframework/quasar#13642] [CLI - Capacitor / Cordova modes]
Hi!
Not sure if this is the right place to post, but does anyone know of any apps made with Quasar for the iPhone? 
I'm wondering if Quasar is good enough, so I want to test what someone has made.

**Comment by MilosPaunovic** [comment]
[https://github.com/quasarframework/quasar-awesome#projects-using-quasar](https://github.com/quasarframework/quasar-awesome#projects-using-quasar)

**Comment by suleimanas** [comment]
Yes, Quasar is more than good enough 
this is one: https://www.fudget.com/
made by @dannyconnell he is the one who introduced Quasar to me
